src/components/icon/icon.tsx

```diff
@@ -146,11 +151,11 @@ export class Icon {
   }
 
   render() {
-    const { iconName, el, inheritedAttributes } = this;
+    const { inheritedAttributes } = this;
```

Comment:
```suggestion
    const { iconName, inheritedAttributes, flipRtl } = this;
```

If we are accessing a class var multiple times we typically destructure. This will let you access just `flipRtl` instead of `this.flipRtl`